src/hooks/useIsVisible/index.ts

```diff
@@ -0,0 +1 @@
+export { useIsVisible } from './useIsVisible'
```

Comment:
Nit: I'd like us to [stop using barrel files](https://tkdodo.eu/blog/please-stop-using-barrel-files).

Reply:
Make sense to me. Which approach do you prefer:
1. `utils/useIsVisible/useIsVisible.ts`,
2. `utils/useIsVisible/index.ts`,
3. `utils/useIsVisible.ts`?

I would go with the last one, and when we need to split logic into more files, then [2]. What do you think?
I would make such changes in batch after merging this and eslint related pull requests.

Reply:
I like the third approach. 

As soon as we get an unreasonable number of files at the same level, then we can group a few into a folder.